### PR TITLE
Fix chmod and exclude tests/benchmark in build

### PR DESCRIPTION
**Pull request by shadow-42**
Faster build process

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -148,7 +148,7 @@ AC_ARG_ENABLE([hardening],
   [AS_HELP_STRING([--disable-hardening],
   [do not attempt to harden the resulting executables (default is to harden)])],
   [use_hardening=$enableval],
-  [use_hardening=yes])
+  [use_hardening=no])
```
Comment (Tech1k) [inline configure.ac]: Let's keep hardening on, for security.

Comment (shadow-42) [reply configure.ac]: oops... that wasn't what I meant to do.

Comment (shadow-42) [reply configure.ac]: pushing new PR

**Comment by shadow-42** [comment]
new pr

**Pull request closed without merging**